drv/Map.drv

```diff
@@ -142,7 +210,459 @@ public interface MAP KEY_VALUE_GENERIC extends FUNCTION KEY_VALUE_GENERIC, Map<K
 
 	@Deprecated
 	@Override
-	boolean containsValue(Object value);
+	default boolean containsValue(Object ov) {
+		if (!(ov instanceof VALUE_CLASS)) return false;
+		return containsValue(VALUE_OBJ2TYPE(ov));
+	}
+
+#endif
+
+	// Defaultable methods
+
+#if KEYS_PRIMITIVE || VALUES_PRIMITIVE
+
+	/** Returns the value to which the specified key is mapped, or the {@code defaultValue} if this map contains no mapping for the key.
+	 *
+	 * @param key the key.
+	 * @param defaultValue the default mapping of the key.
+	 *
+	 * @return the value to which the specified key is mapped, or the {@code defaultValue} if this map contains no mapping for the key.
+	 *
+	 * @see java.util.Map#getOrDefault(Object, Object)
+	 * @since 8.0.0
+	 */
+
+	default VALUE_GENERIC_TYPE GET_VALUE_OR_DEFAULT(KEY_TYPE key, VALUE_GENERIC_TYPE defaultValue) {
+#if VALUES_PRIMITIVE
+		return containsKey(key) ? GET_VALUE(key) : defaultValue;
+#else
+		V v;
+		return (v = get(key)) != null || containsKey(key) ? v : defaultValue;
+#endif
+	}
+
```

Comment:
I am strongly against something like `if (v != drv) return v;`. In the documentation of Function, it is written that `null` should always be used to signal absence of a key when `get` is called. If we now introduce such a check we probably can create some weird behavior.

The weirdness of `null` is something most people agree on I think, but we should stick with it.